infra/loki/loki-config.yaml

```diff
@@ -0,0 +1,49 @@
+auth_enabled: false
```

Comment:
Is this file some default config or very specific to our use case? If that's from some documentation maybe it's worth adding some link as a comment to this file? I feel like most of us are not that familiar with this config.

Reply:
Comparing this config file from one of our projects, this looks fine. Though, please add the source if it wasn't written by hand.

Reply:
To be fair it was mostly generated from Perplexity 😅, so I could add a comment that it was generated this way but I don't think it would be very beneficial.